openslides/users/projector.py

```diff
@@ -16,5 +16,16 @@ def user_slide(all_data: AllData, element: Dict[str, Any]) -> Dict[str, Any]:
     return {"error": "TODO"}
 
 
+def get_user_short_name(all_data: AllData, user_id: int) -> str:
```

Comment:
please use always the full_name for submitters.

Reply:
Can you tell ne what data is in the full_name?

Is the short_name needed at any place on the projector or can I remove this function?

Reply:
You can remove "no." because we can't translate it.

Reply:
I am not sure if I understand you correctly. If not, you can either tell me again what to change, or you merge this pr and fix it in another pr

Reply:
Ok for me. Thx.